### PR TITLE
add alpine images to realpage/php

### PR DESCRIPTION
**Pull request by millidavids**

### CODE REVIEW

**Comment by bkuhl** [comment]
I'd be a fan of switching primarily to Alpine, but maintaining a Jesse branch in the event we need to hotfix an existing app.

File: 7.0/alpine/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+# Locally docker mounts files as user 1000
+# Adding this user to www-data allows php
+# to write to these files
+RUN usermod -u 1000 www-data && \
+    usermod -G staff www-data
+
+# fix permissions on a mac
+RUN sed -ri 's/^www-data:x:82:82:/www-data:x:1000:50:/' /etc/passwd \
+    && sed -ri 's/^www-data:x:82:/www-data:x:50:/' /etc/group
```
Comment (bkuhl) [inline 7.0/alpine/Dockerfile]: With the new docker beta client, we could probably get rid of these permissions adjustments.

Comment (millidavids) [inline 7.0/alpine/Dockerfile]: Noted

File: 7.0/alpine/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+FROM php:7.0.5-alpine
+MAINTAINER Foundation DevOps foundation-devops@realpage.com
+LABEL version="1.0.0-beta" \
```
Comment (bkuhl) [inline 7.0/alpine/Dockerfile]: I think we agreed on other projects to eliminate these "version" tags.  We should probably remove it here too

File: 7.0/alpine/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+      com.realpage.php.version="7.0.5" \
+      com.realpage.php.type="cli"
+
+# Locally docker mounts files as user 1000
+# Adding this user to www-data allows php
+# to write to these files
+RUN usermod -u 1000 www-data && \
+    usermod -G staff www-data
```
Comment (bkuhl) [inline 7.0/alpine/Dockerfile]: We should probably remove this portion too... I think this was the permissions fix for PC

**Comment by bkuhl** [comment]
👍 Looks good here.  We'll probably need fpm-alpine as well

**Comment by jlaswell** [comment]
https://travis-ci.org/realpage/php/jobs/142773511#L3234 Seems to be the issue. I think the hyphens may still be throwing off generating a list of tags.

**Comment by jlaswell** [comment]
👍

**Pull request merged**